src/components/pages/profile-overview/profile-overview.tsx

```diff
@@ -43,49 +41,49 @@ const ProfileOverview: React.FC<{
           {!data ? (
             <SkeletonTextLine fontSize={1.6} />
           ) : (
-            `${data.userInfo.profile.name} ${data.userInfo.profile.middle_name} ${data.userInfo.profile.family_name}`
+            `${data.profileUserInfo.name} ${data.profileUserInfo.middle_name} ${data.profileUserInfo.family_name}`
```

Comment:
```suggestion
            `${data.profileUserInfo.name}${data.profileUserInfo.middle_name ? " " + data.profileUserInfo.middle_name :  ""} ${data.profileUserInfo.family_name}`
```
Because for now we always have a middlename but it may not always be the case.

Reply:
Well, I'd say this is the case for each value, no? As the type is not defined yet, I did not bother checking them first. What do you think?

Reply:
I approved the PR, so do as you want 😁
We will be forced to check this kind of stuff at some point anyway